README.md

```diff
@@ -1,14 +1,66 @@
-# Project
+# Cosmos TLS Scanner
 
-> This repo has been populated by an initial template to help get you started. Please
-> make sure to update the content to build a great experience for community-building.
+This utility can be used to detect the supported TLS version for a given CosmosDB account.
 
-As the maintainer of this project, please make a few updates:
+Note that this utility would only be able to test the account against TLS versions supported by the client system running the script.
 
-- Improving this README.MD file to provide a great experience
-- Updating SUPPORT.MD with content about this project's support experience
-- Understanding the security reporting process in SECURITY.MD
-- Remove this section from the README
+Supported TLS version:
+- SSL V2
+- SSL V3
+- TLS V1
+- TLS V1.1
+- TLS V1.2
+- TLS V1.3
+
+## Getting started
+
+### Prerequisites
+- [Python 3.5.3+][python]
+- Install requirements from requirements.txt
+- A CosmosDB SQL API account to test for supported TLS versions.
+    - Account endpoint like https://myaccount.documents.azure.com:443/
+    - Any of the read-write or read-only [primary keys][primary_keys]
+    - Optional database and collection name.
+
+### Running the TLS scanner util
+
+Installing requirements.txt
+```bash
+pip install -r requirements.txt
+```
+
+Run the utility with a given CosmosDB endpoint as follows
+```bash
+python3 cosmos_tls_scanner.py --endpoint https://myaccount.documents.azure.com:443/ --authorization-key khYANAIiAl12n...==
+```
+Or additionally provide the database and collection name to test the TLS version against
+```bash
+python3 cosmos_tls_scanner.py --endpoint https://myaccount.documents.azure.com:443/ --authorization-key khYANAIiAl12n...== --database-name MyDatabase --collection-name MyCollection
+```
+
+### Interpreting results
+
+On running the script, we see results that look like this
+```
+Endpoint: [https://myaccount.documents.azure.com:443/]
+Database name: [MyDatabase], Collection name: [MyCollection]
+
+SSL/TLS Version Support
+SSL V2 (not supported by Cosmos DB) : Not supported by client
+SSL V3 (not supported by Cosmos DB) : Not supported by client
+TLS V1   Basic Query : Not supported
+TLS V1.1 Basic Query : Supported
+TLS V1.2 Basic Query : Supported
+TLS V1.3 : Not supported by client
+```
+
```

Comment:
"This example basically says that TLS V1 is not accepted by Cos..." to "This result indicates that TLS V1 is not accepted by Cos..."

"This could happen if python" to "This can also happen if python"